packages/dvilela/connections/twikit/connection.py

```diff
@@ -107,13 +107,11 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:  # pragma: no cover
         self.username = self.configuration.config.get("twikit_username")
         self.email = self.configuration.config.get("twikit_email")
         self.password = self.configuration.config.get("twikit_password")
-        cookies_str = self.configuration.config.get("twikit_cookies")
-        self.cookies_path = Path(
-            self.configuration.config.get(
-                "twikit_cookies_path", "/tmp/twikit_cookies.json"  # nosec
-            )
+        self.cookies = self.configuration.config.get("twikit_cookies")
+        self.cookies_path = (
+            Path(self.configuration.config.get("store_path", "/tmp"))
+            / f"twikit_cookies_{self.username}.json"
```

Comment:
On the current approach cookies of user, e.g., johndoe, are stored on
`.operate/services/sc-1d5f03fa-6559-493e-a9b0-f12c52ad3adf/persistent_data/johndoe/twikit_cookies.json`
Consider if it's worth continuing this approach, especially for existing services.

Reply:
that is STORE_PATH / johndoe / twikit_cookies.json

Reply:
Will store_path point to `operate/services/sc-1d5f03fa-6559-493e-a9b0-f12c52ad3adf/persistent_data/`?
So you mean that I should add the username as a subdir right?

Reply:
Yes